File: VB/Q139656/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors.Controls
Imports DevExpress.XtraBars.Docking

Namespace Q139656
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			For i As Integer = 0 To Screen.AllScreens.Length - 1
				radioGroup1.Properties.Items.Add(New RadioGroupItem(i, "Screen #" & i.ToString()))
			Next i
		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			dockPanel1.MoveToScreen(New Point(100, 100), radioGroup1.SelectedIndex)
		End Sub
	End Class
End Namespace

File: CS/Q139656/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors.Controls;
using DevExpress.XtraBars.Docking;

namespace Q139656 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            for (int i = 0; i < Screen.AllScreens.Length; i++)
                radioGroup1.Properties.Items.Add(new RadioGroupItem(i, "Screen #" + i.ToString()));
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            dockPanel1.MoveToScreen(new Point(100, 100), radioGroup1.SelectedIndex);
        }
    }
}